### DISCUSSION TITLE
The newest firmware cannot detect motors

### DISCUSSION THREAD

**Discussion by Potato-eater** [pybricks/support#1133] [Q&A] [answered]
I had pybricks version 3.3.0b7 installed on my robot, and it could not detect any motors. 
I checked and repluged the wires again, and it was still like that. So I installed an older firmware (3.3.0b5) onto the robot, and it started working again. Was there some changes in the newest firmware that caused this?
![image](https://github.com/pybricks/support/assets/95858994/44afd818-547d-4bd1-8a47-ab43b1e25c23)

**Comment by laurensvalk** [comment, marked as answer]
Yes, see https://github.com/pybricks/support/issues/1131 for details.
